src/styles/card.css

```diff
@@ -0,0 +1,115 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  flex-grow: 1;
+}
+
+.card__image {
+  position: relative;
+  left: calc(50% - 160px / 2);
+}
+
+.card__imac {
+  display: flex;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-top: 40px;
+}
+
+.title {
+  display: flex;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-dark);
+}
+
+.card__code {
+  margin-top: 4px;
+}
+
+.code {
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--color-light);
+  line-height: 14px;
+}
+
+.card__reviews-block {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-light);
+}
+
+/* #region card__price-block */
+.card__price-block {
+  margin-top: 24px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.price {
+  font-weight: 400;
+  font-size: 12px;
+  color: var(--color-light);
+  line-height: 18px;
+}
+
+.value {
+  font-weight: 700;
+  font-size: 16px;
+  color: var(--color-dark);
+  line-height: 18px;
+}
+/* #endregion */
+
+/* #region card__buy_button */
```

Comment:
```suggestion
```
you already have divided styles into separate files, I think it would be better to remove these comments. But that is not a mistake, just a proposition